src/WebApp/Pages/Shared/CategoryHeader.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿@using Codidact.Core.Application.Categories.Queries.ShortCategoriesListQuery
+@inject ShortCategoriesListQuery query
```

Comment:
Update Co-Design to version 0.6.0 and use the Category Header component:

https://design.codidact.org/specific/category-header.html

Reply:
[![Image from Gyazo](https://i.gyazo.com/298edbee87181166fb70d95a4cc31956.png)](https://gyazo.com/298edbee87181166fb70d95a4cc31956)
Not sure where to get the guidelines for this. I added the category but not sure about it all

Reply:
The idea was to have two headers:

- a site-header with the community name and all community-wide functions and the authentification details (current user icon, sign out link, ...)
- a category-tabination and header, which allows users to choose a category and will contain all category-wide functions (new post, list all posts, list all tags)

See this wireframe for the idea:

![wireframe that shows the two header elements](https://user-images.githubusercontent.com/21335202/79564430-e998ee00-80ae-11ea-9687-eb993cc2ab7c.png)

(source: https://www.figma.com/file/X58CjsqsxmSLCUwd0JS5Xt/process-global?node-id=0%3A1)

This requirement has been implemented in Co-Design with two components:

- [Header](https://design.codidact.org/components.header.html)
- [Category Header](https://design.codidact.org/specific/category-header.html)

So the Header should contain:

- **Codidact** (to be replaced later by the community name)
- Categories entry (or tbd later)
- Users list (or tbd later)
- Current user view / Authentification buttons

The category header should contain:

- category tabs (as in your PR now)
- Questions link for the current category
- Tabs link for the current category (or tbd later)
- Search link for the current category (or tbd later)
- New Question link for the current category

I don't think we need "on GitHub" there, as this will probably go into a footer later and doesn't seem to be useful right now, when it isn't hosted anywhere for others to see.

Reply:
Alright so:
[![Image from Gyazo](https://i.gyazo.com/b460cfd32c2aab06e68b1cf51bd16e37.png)](https://gyazo.com/b460cfd32c2aab06e68b1cf51bd16e37)

Reply:
Yeah. Looks good. The color will be changeable by category, but that doesn't need to be done now.

Reply:
Alright so this is resolved